docker/konduit-python/start-server.py

```diff
@@ -0,0 +1,115 @@
+import random
+import re
+import sys
+import os
+import numpy as np
+import time
+
+from konduit import ServingConfig, PythonPipelineStep
+from konduit import InferenceConfiguration
+from konduit.server import Server
+from konduit.client import Client
+from konduit.utils import is_port_in_use
+
+
+def cleanup(value, key):
+    return value.replace("<{}:".format(key), "").replace(":>", "")
+
+
+def separate(value, delimiter):
+    separated = value.split(delimiter)
+    return separated[0].strip(), separated[1].strip()
+
+
+def get_individuals(read_input, key):
+    matches = re.findall('<{}:.*:>'.format(key), read_input)
+
+    if "rest" in key:
+        # Since only one rest input and output type makes sense
+        ret = cleanup(matches[0], key)
+    else:
+        ret = []
+        for match in matches:
+            ret.extend([separate(split, "|") for split in cleanup(match, key).split(",")])
+
+    return ret
+
+
+def schema_type(python_type):
+    return {
+        "NDARRAY": "NDArray",
+        "BOOL": "Boolean",
+        "STR": "String",
+        "INT": "Integer",
+        "FLOAT": "Float"
+    }.get(python_type, "NDArray")
+
+
+with open('script.py', 'r') as script_file:
+    content = script_file.read()
+
+    inputs = get_individuals(content, "in")
+    outputs = get_individuals(content, "out")
+
+    rest_input_type = get_individuals(content, "rest_in")
+    rest_output_type = get_individuals(content, "rest_out")
+
+    # ----------------------------------------------------------------------------------------
+
+    port = random.randint(1000, 65535)
+
+    serving_config = ServingConfig(http_port=port,
+                                   input_data_type=rest_input_type,
+                                   output_data_type=rest_output_type,
+                                   log_timings=True)
+
+    pythonConfig = {
```

Comment:
Just of note is you can use a to json utility present right in konduit-serving python: https://github.com/KonduitAI/konduit-serving/blob/master/python/konduit/json_utils.py